.claude/agents/issue-fix.md

```
@@ -16,35 +16,38 @@ You are a specialized agent for debugging and fixing issues in the Leverage Refe
-2. **Get current AWS provider documentation:**
+2. **Get current provider documentation:**
    ```
-   mcp__terraform-mcp__SearchAwsccProviderDocs(
-     asset_name="awscc_<resource>",
-     asset_type="resource"
+   mcp__plugin_context7_context7__resolve-library-id(
+     libraryName="Terraform AWS Provider",
+     query="aws_<resource> / awscc_<resource> arguments"
    )
    ```
    ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)